tensorflow/python/data/experimental/ops/threading_options.py

```diff
@@ -52,6 +54,15 @@ class ThreadingOptions(options.OptionsBase):
       "The value 0 can be used to indicate that the threadpool size should be "
       "determined at runtime based on the number of available CPU cores.")
 
+  def _get_option_attrs(self):
+    return ["max_intra_op_parallelism", "private_threadpool_size"]
+
+  def _has_non_default_values(self):
+    for attr in self._get_option_attrs():
+      if object.__getattribute__(self, attr) is not None:
```

Comment:
use `getattr` instead of `object.__getattribute__`

Reply:
addressed in the commit